tests/integration/resource_manager/test_engine.py

```diff
@@ -42,10 +42,11 @@ def test_create_start_stop_engine(
 ParamValue = namedtuple("ParamValue", "set expected")
 ENGINE_UPDATE_PARAMS = {
     # commented parameters are not available yet
-    #    "scale": ParamValue(23, 23),
-    #    "spec": ParamValue("B1", "B1"),
+    "scale": ParamValue(23, 23),
```

Comment:
We're testing on x23? Should we use something $$ safer, in case it's started for some reason?

Reply:
It's never started so we should be safe

Reply:
I can reduce the size though anyway